compio-driver/src/lib.rs

```diff
@@ -300,6 +300,25 @@ impl Proactor {
         }
     }
 
+    /// Get the pushed operations from the completion entries.
+    ///
+    /// # Panics
+    /// This function will panic if the requested operation has not been
+    /// completed.
+    pub fn pop_with_flags<T>(
```

Comment:
We can just unify the two pop functions, I think.

Reply:
the `pop` can call `pop_with_flags`

but I think maybe we can still keep the origin `pop`, there are too many changes if replace all `pop` with `pop_with_flags`

Reply:
> there are too many changes

Not many locations call `pop`.

Reply:
done